alicloud/resource_alicloud_cs_swarm.go

```diff
@@ -96,6 +96,11 @@ func resourceAlicloudCSSwarm() *schema.Resource {
 					return old != ""
 				},
 			},
+			"need_slb": &schema.Schema{
```

Comment:
Please output this parameter in the read method

This parameter can not be modified and please set it to forcenew

Reply:
Is this parameter really needed in the read method? If `need_slb` was set to false then slb_id will be empty. So this seems kind of redundant to me?